nevergrad/functions/unitcommitment/core.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import nevergrad as ng
+from ..base import ExperimentFunction
+
+
+class UnitCommitmentProblem(ExperimentFunction):
+    """Model that uses conventional implementation for semi-continuous variables
+    The model is adopted from Pyomo model 1: Conventional implementation for semi-continuous variables
+    (https://jckantor.github.io/ND-Pyomo-Cookbook/04.06-Unit-Commitment.html)
+    The constraints are added to the objective with a heavy penalty for violation.
+
+    Parameters
+    ----------
+    num_timepoints: int
+        number of time points
+    num_generators: int
+        number of generators
+    penalty_weight: float
+        weight to penalize for violation of constraints
+    """
+
+    def __init__(
+        self,
+        problem_name="semi-continuous",
+        num_timepoints: int = 13,
+        num_generators: int = 3,
+        penalty_weight: float = 10000,
+    ) -> None:
+        if problem_name not in ["semi-continuous"]:
+            raise NotImplementedError
+
+        params = {x: y for x, y in locals().items() if x not in ["self", "__class__"]}  # for copying
```

Comment:
```suggestion
```
The CI states that it's not used (and recent changes make it not needed anymore for copying